lively.merger/README.md

```diff
@@ -1,3 +1,24 @@
 # lively.merger
 
-No description for package lively.merger yet.
+## Entry Point
+
+The entry point is the merger class. In this the different functionality for merging morphs is described.
+This falls back on the 3-way-merge class. In this the actual merge happens on object level.
+For a 3-way-merge we need 3 objects. These are compared on the basis of their keys. To merge morphs we use the propertiesAndPropertySettings method.
+So we merge only the properties of the morphs and then merge them to a new morph or overwrite the properties at one of the original morphs.
```

Comment:
Content is good, I'd just adapt some of the phrasing and add links to the respective files

```suggestion
To merge the morphs, we extract their properties using the `propertiesAndPropertySettings()` method. These can then be handled as normal objects and passed into the [3-way-merger](./3-way-merger.js). The resulting merged properties are then either used to create a new, merged morph, or applied to one of the input morphs to allow 'merge into' functionality
```